examples/plugins/src/lighthouse/src/lighthouse.plugin.integration.test.ts

```diff
@@ -0,0 +1,144 @@
+      '--no-verbose',
+      '--output="json"',
+      '--output-path="lighthouse-report.json"',
+      '--onlyAudits="first-contentful-paint"',
+      '--onlyAudits="largest-contentful-paint"',
+      '--onlyAudits="speed-index"',
+      '--onlyAudits="total-blocking-time"',
+      '--onlyAudits="cumulative-layout-shift"',
+      '--onlyAudits="server-response-time"',
+      '--onlyAudits="interactive"',
+      '--chrome-flags="--headless=new"',
+    ]);
+  });
+
+  it('should parse options for headless by default to "new"', async () => {
+    const pluginConfig = await create({
+      url: LIGHTHOUSE_URL,
+    });
+    expect(pluginConfig.runner.args).toEqual(
+      expect.arrayContaining(['--chrome-flags="--headless=new"']),
+    );
+  });
+
+  it('should parse options for headless to new if true is given', async () => {
+    const pluginConfig = await create({
+      url: LIGHTHOUSE_URL,
+      headless: true,
+    });
+    expect(pluginConfig.runner.args).toEqual(
+      expect.arrayContaining(['--chrome-flags="--headless=new"']),
+    );
+  });
+
+  it('should parse options for headless to new if false is given', async () => {
+    const pluginConfig = await create({
+      url: LIGHTHOUSE_URL,
+      headless: false,
+    });
+    expect(pluginConfig.runner.args).toEqual(
+      expect.not.arrayContaining(['--chrome-flags="--headless=new"']),
+    );
+  });
+
+  it('should parse options for userDataDir correctly', async () => {
+    const pluginConfig = await create({
+      url: LIGHTHOUSE_URL,
+      userDataDir: 'test',
+    });
+    expect(pluginConfig.runner.args).toEqual(
+      expect.arrayContaining([
+        '--chrome-flags="--headless=new --user-data-dir=test"',
+      ]),
+    );
+  });
+
+  it('should return PluginConfig that executes correctly', async () => {
+    const pluginConfig = await create({ url: LIGHTHOUSE_URL });
+    await expect(executePlugin(pluginConfig)).resolves.toMatchObject(
+      expect.objectContaining({
+        slug,
+        title: 'Lighthouse',
+        description: 'Chrome lighthouse CLI as code-pushup plugin',
+        duration: expect.any(Number),
+        date: expect.any(String),
+        audits: expect.any(Array),
+        groups: expect.any(Array),
+      }),
+    );
+  });
+
+  it('should use onlyAudits', async () => {
+    const pluginConfig = await create({
+      url: LIGHTHOUSE_URL,
+      onlyAudits: 'largest-contentful-paint',
+    });
+    expect(pluginConfig.runner.args).toEqual(
+      expect.arrayContaining(['--onlyAudits="largest-contentful-paint"']),
+    );
+    const { audits: auditOutputs } = await executePlugin(pluginConfig);
+
+    expect(auditOutputs).toHaveLength(1);
+  });
+}, 30_000);
+
+describe('lighthouse-audits-export', () => {
+  it.each(audits)('should be a valid audit meta info', audit => {
+    expect(() => auditSchema.parse(audit)).not.toThrow();
+  });
+});
+
+describe('lighthouse-corePerfGroupRefs-export', () => {
+  it.each(corePerfGroupRefs)(
```

Comment:
Similarly to the audits, the group reference audit should be added to the test description, otherwise multiple test cases of the same name will be logged as output.

Reply:
done